lib/dogma.ex

```diff
@@ -10,13 +10,33 @@ defmodule Dogma do
   alias Dogma.Rules
   alias Dogma.ScriptSources
 
-  def run(dir_path \\ "") do
-    dir_path
+  @formatters %{
+    "simple" => Formatter.Simple,
+    "emacs"  => Formatter.Emacs
+  }
+
+  def run(argv) do
+    argv
+    |> parse_args
+    |> process
+  end
+
+  def parse_args(argv) do
```

Comment:
I thought about pulling it out into it's own module, but it didn't seem like it would be necessary until there were more configuration options.